docs/content/operator/gas-station/deployment/deployment.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Deployment Guide
+description: Learn how to deploy the IOTA Gas Station for testing and production environments.
+teams:
+  - iotaledger/identity
+tags:
+  - gas-station
+  - setup
+---
+
+
+The IOTA Gas Station is distributed as two Docker images:
+
+1. [iotaledger/gas-station-tool](https://hub.docker.com/r/iotaledger/gas-station-tool) – A utility tool for:
+
+   * Generating configuration files
+   * Performing health checks
+   * Running benchmarks
+
+2. [iotaledger/gas-station](https://hub.docker.com/r/iotaledger/gas-station) – The Gas Station server with the exposed **HTTP API**.
+
+---
+
+## **Test Deployment**
```

Comment:
I wonder if we should link to the repo for the following instructions in case they change or are extended

Reply:
I've added the `info` section.